### DISCUSSION TITLE
Can I install older version at my Ubuntu 20.04?

### DISCUSSION THREAD

**Discussion by DavidTbilisi** [RHVoice/RHVoice#289] [Q&A]
I have Ubuntu 20.04 and I want to use Georgian language (Natia voice) which is not at latest version of RHVoice.

**Comment by alex19EP** [comment]
are you using a PPA or compiling from source?

**Reply by DavidTbilisi** [reply]
I'm using PPA

**Reply by alex19EP** [reply]
ok then you should ask @linvinus he is a PPA maintainer.
or you can compile rhvoice your self.

**Reply by DavidTbilisi** [reply]
Okay. Thanks. 
and can you give me a hint how can I figur out which version have disired for me voice?

**Reply by alex19EP** [reply]
latest master have this voice.

**Reply by DavidTbilisi** [reply]
Am I doing everithing right? 
This is my command for RHVoice read text in 3 languages.
```bash 
echo ტესტი тест test | RHVoice-client --speakers Natia+anna+CLB | aplay
``` 
because it's reading Russian, English but not reading Georgian :(

**Reply by alex19EP** [reply]
> because it's reading Russian, English but not reading Georgian :(

to be honest, I'm not sure if you can read Georgian this way. also not sure if you can combine three voices at the same time.
maybe @Olga-Yakovleva knows more.

**Reply by DavidTbilisi** [reply]
Okay Thanks anyway @alex19EP for your responsiveness

**Comment by beqabeqa473** [comment]
you can read three languages with RHVoice-test utility.

voice profiles are constructed with + (plus) signh.

i don't think ppa have this voice included because of license restrictions.

if you agree with license terms, you can compile RHVoice yourself and
while compiling pass flag to scons to install georgian language.

On 4/20/21, David ***@***.***> wrote:
> Okay Thanks anyway @alex19EP for your responsiveness
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/RHVoice/RHVoice/discussions/289#discussioncomment-632608


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Reply by DavidTbilisi** [reply]
@beqabeqa473 Can you show me an example of installing command?
I have tryied:
```bash
scons install --languages georgian,russian,english
```

**Reply by DavidTbilisi** [reply]
I think I have found answer [in this commit](https://github.com/RHVoice/RHVoice/commit/92c9eecb7def68c82f979eaf2f85d3d54879df07), but it doesn't solve my problem
